### ISSUE TITLE
Section id has double-double quotes.

### ISSUE THREAD

**Issue by anitagraham** [linchpin/mesh#253]
Trying to target a mesh section id, I notice that it has double-double quotes.  

```
<section class="mesh_section post-1908 type-mesh_section status-publish hentry" id="&quot;summary&quot;">
```

The quotes are not visible in the UI.

![image](https://user-images.githubusercontent.com/397118/48040710-0d1af200-e1b5-11e8-8523-089ca376afc8.png)

**Comment by aaronware** [comment]
This is resolved in 1.2.5.4 which is deploying momentarily

**Comment by anitagraham** [comment]
Thanks - works perfectly. Sorry for the delay in responding - computer was upgraded and  took a day or so to recover.
